### ISSUE TITLE
Enable concurrent actions

### ISSUE THREAD

**Issue by victortokaito** [osmosis-labs/osmosis-frontend#817]
### Problem
Users have to wait for a transaction to process before continuing on to perform different LP actions:

https://user-images.githubusercontent.com/39136683/191379855-b9affdab-7c60-4ccb-8788-f337745814f8.mov

### Proposal
Is it possible to enable concurrent actions so that users can submit multiple transactions? It'd be nice to have multiple "add liquidity" or swap actions run in parallel so users don't have to wait for each transaction to complete.

**Comment by jonator** [comment]
I don't think keplr allows multiple txs to be sent at once

**Comment by victortokaito** [comment]
@delivan If possible, what would implementation look like to make this happen? It might vastly improve high frequency traders' quality of life -> more volume
